### ISSUE TITLE
Blending Data Points 

### ISSUE THREAD

**Issue by n1shamin** [PardeeCenterDU/IFs-Issues-Tracking#332]
I was trying to blend certain data points (ie Brazil only the years 1960-1989) but when I tried to do that I received this error: 
![image](https://github.com/user-attachments/assets/bf9ef07f-ab3b-4809-875e-341d6227650a)

**Comment by PardeeCenterIFs** [comment]
Can you include the file that you're using for this issue.

**Comment by n1shamin** [comment]
[IFsDataImportwdi.zip](https://github.com/user-attachments/files/16630919/IFsDataImportwdi.zip)

**Comment by PardeeCenterIFs** [comment]
Wow, this file has lots of tables, when I open it, all of them are selected, what next?
Should I just select Brazil, click Blend Data Points and it will try to blend all selected series? 
Or are you trying with a single variable, in which case, which one?

**Comment by n1shamin** [comment]
I have a screen recording the process but the file is too big to attach here. 
In the screen recording I chose VaddMan% and tried to blend the missing data points from Togo. This would be the years 1960-2006. I selected Togo from the Country list and I have tried both Select Missing Years and selected each missing year from the year column. Once I do that I click blend data points and it gives me that above error. If you would like the screen recording I could send it to you.
